### ISSUE TITLE
[Settings> Add a new profile]: The Name property must not include the element's control type.

### ISSUE THREAD

**Issue by ghost** [microsoft/terminal#11156]
### Windows Terminal version (or Windows build number)

1.10.2383.0

### Other Software

**Test Environment:**
OS: Windows 11 Version 21H2 (OS Build 22000.168)
Screen Reader: Narrator
Tool: Accessibility Insights for Windows Version 1.1.1689.01

### Steps to reproduce

**Repro Steps:**

1. Open Windows Terminal.
2. Open Settings page using "Ctrl + ',' ".
3. Run Accessibility insights for Window on Settings Add a new Profile tab page and observe the issue.

**User Experience: -> Priority 3**
Screen Reader user are impacted here as control type is announced twice with control name which is confusing.

**Guideline Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

**Attachments:**
[The Name property must not include the element's control type..zip](https://github.com/microsoft/terminal/files/7115865/The.Name.property.must.not.include.the.element.s.control.type.zip)

### Expected Behavior

The Name property must not include the element's control type. For 'New Empty Profile' Screen Reader should announce 'Create new Empty Profile button' and for 'Duplicate' Screen Reader should announce 'Duplicate button'. 

### Actual Behavior

The Name property has the element's control type. For 'New Empty Profile' button Screen Reader is announcing 'Create new button button' and For 'Duplicate' button Screen Reader is announcing 'Duplicate button button'. 

**Note Issue:** The Name must not include the same text as the LocalizedControlType.

**Comment by v-rpundir** [comment]
Issue repro in Windows Terminal Preview Version: 1.12.3472.0

**Actual:** For 'New Empty Profile' Screen Reader is announcing 'Create New button button' and For 'Duplicate' control Screen Reader is announcing 'Duplicate button button'.

**Expected:** For 'New Empty Profile' Screen Reader should announce 'New Empty Profile button' and for 'Duplicate' control Screen Reader should announce 'Duplicate Profile button'.

**Comment by carlos-zamora** [comment]
Fixed by #12324 a while back. Should be available on versions 1.14.1432 or greater
